### ISSUE TITLE
Search Capabilities

### ISSUE THREAD

**Issue by iamsmruti** [processing/p5.js-showcase#6]
There should be a search along with a filter to improve the accessibility of the application.

Benefits of adding a search capability:
- people would be able to search through tags and get the projects that is being made using a specific function or library
- easier to navigate and discover related projects

**Comment by akshaybhasin4** [comment]
Can you assign this issue to me? I can add this feature

**Comment by akshaybhasin4** [comment]
Hey @iamsmruti,
I have created PR for this, could you please review it and give feedback?
Thank You.
